static/js/compose.js

```diff
@@ -1074,6 +1074,21 @@ exports.initialize = function () {
         })
     );
 
+    $("#compose-textarea").focus(function () {
+        var msg_type = compose_state.get_message_type();
+        var stream = $('#stream_message_recipient_stream').val();
+        var topic = $('#stream_message_recipient_topic').val();
+        var pm_recipients = compose_pm_pill.get_emails();
```

Comment:
What's your take on directly assigning these values in the `opts` object compared to creating these extra assignments?

Reply:
I didn't really give it much thought in this particular case, but I think I've seen/done it (i.e., similar objects) the same way in other parts of our code.
Do you think I should just assign the value directly? It's the same either way if that is what you're asking.

Reply:
Right, I only mention this because the variables are referenced only once, and the `opts` keys adequately describe the values.  My thinking was directly assigning the values might make things a bit cleaner.
> Do you think I should just assign the values directly?

Not sure, it's pretty minor, maybe we should wait for another's opinion :)

Reply:
That makes. I guess I just didn't think of it at the time. Made the change to directly assign the values to `opts` now. Thanks for pointing it out, @whoodes